.gitignore

```diff
@@ -18,7 +18,7 @@
 !/tmp/pids/
 !/tmp/pids/.keep
 
-
+.DS_Store
```

Comment:
P.S. I strongly recommend adding this to your global git ignore.

```
$ echo .DS_Store >> ~/.gitignore_global
$ git config --global core.excludesfile ~/.gitignore_global
```

Reply:
I wasn't aware Git had a global gitignore. TIL